models/page_views/scratch/bigquery/snowplow_web_page_views_this_run.sql

```diff
@@ -116,7 +116,7 @@ from (
 )
 
 {% if var("snowplow__ua_bot_filter", true) %}
-   where not regexp_contains(ev.useragent, '(bot|crawl|slurp|spider|archiv|spinn|sniff|seo|audit|survey|pingdom|worm|capture|(browser|screen)shots|analyz|index|thumb|check|facebook|PingdomBot|PhantomJS|YandexBot|Twitterbot|a_archiver|facebookexternalhit|Bingbot|BingPreview|Googlebot|Baiduspider|360(Spider|User-agent)|semalt)')
+   {{ filter_bots() }}
 {% endif %}
```

Comment:
🔧 
To increase consistency across the warehouses and avoid issues in the future, would it be worth writing this as 
```sql
where 
   1=1
{% if var("snowplow__ua_bot_filter", true) %}
   {{ filter_bots() }}
{% endif %}
```

and then all of the macros start with `and`?

Reply:
makes sense, will push it through again with the above change as well.